src/tests/ftest/server/replay.py

```diff
@@ -0,0 +1,66 @@
+"""
+  (C) Copyright 2019-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from apricot import TestWithServers
+from ior_utils import run_ior
+from job_manager_utils import get_job_manager
+from test_utils_pool import add_pool
+
+
+class ReplayTests(TestWithServers):
```

Comment:
I don't think we have a formal pattern, but how do you feel about putting the directory name in the class name? E.g.
```suggestion
class ServerReplayTest(TestWithServers):
```

Reply:
Yeah, I don't disagree with that. I'm thinking more for the sake of guaranteeing unique test method names, but that's a topic for another time